src/svgcontext.ts

```diff
@@ -561,7 +566,9 @@ export class SVGContext implements RenderContext {
 
       const sweep = !antiClockwise;
 
-      this.path += `M${x0} ${y0} A${radius} ${radius} 0 ${+large} ${+sweep} ${x1} ${y1}`;
+      this.path += `M${round3(x0)} ${round3(y0)} A${round3(radius)} ${round3(radius)} 0 ${+large} ${+sweep} ${round3(
+        x1
+      )} ${round3(y1)}`;
       this.pen.x = x1;
```

Comment:
better to alias `const r3 = round3`; and fit on one line so as not to introduce newlines into the path variable.

(I wonder if just want to do that in the function name so people have to look up what it is instead of assuming it'll return a number not a string?)

Reply:
+1 to alias